Automattic/wp-calypso
# Launchpad: Enable to use feature flag by setup flow and add flag to the start-writing flow

Related to https://github.com/Automattic/wp-calypso/issues/86647

## Proposed Changes
* Enable to use feature flag by setup flow 
* Add feature flag to the `start-writing` flow
* Use a global feature flag to turn off all flows
* The code will check the flags `"launchpad/new-task-definition-parser` and ` "launchpad/new-task-definition-parser/[FLOW_NAME]'

## Testing Instructions
- Load  the setup/free flow  (E.g. /setup/free/launchpad?siteSlug=gabrielcaires-free-flow.wordpress.com)
- Check if there is messages on console log lke the following ones.
``` 
Using new task definition parser {taskId: 'plan_selected', flowId: 'free'}
Using new task definition parser {taskId: 'setup_free', flowId: 'free'}
Using new task definition parser {taskId: 'design_selected', flowId: 'free'}
Using new task definition parser {taskId: 'domain_upsell', flowId: 'free'}
Using new task definition parser {taskId: 'first_post_published', flowId: 'free'}
Using new task definition parser {taskId: 'design_edited', flowId: 'free'}
Using new task definition parser {taskId: 'site_launched', flowId: 'free'}
``` 

## Pre-merge Checklist
